presentation/backend/CHANGELOG.json

```diff
@@ -829,18 +859,6 @@
       "date": "Tue, 06 Jul 2021 22:08:34 GMT",
       "comments": {}
     },
-    {
-      "version": "2.16.9",
```

Comment:
interesting....

Reply:
yeah don't know how this got here to begin with